### PR TITLE
melt + pivot

### PR DESCRIPTION
**Pull request by sorhawell**
solves #231

### CODE REVIEW

**Comment by sorhawell** [comment]
DataFrame.pivot + DataFrame.melt + tests still missing

File: tests/testthat/_snaps/knitr.new.md
Comment (eitsupi) [inline tests/testthat/_snaps/knitr.new.md:1]: It seems that unnecessary file has been added.

**Review by eitsupi — CHANGES_REQUESTED** [review]

File: tests/testthat/_snaps/knitr.new.md
Comment (sorhawell) [reply tests/testthat/_snaps/knitr.new.md:1]: Oh, thanks :)

**Review by eitsupi — APPROVED** [review]
Thanks!

**Pull request merged**